Sources/Components/HorizontalSlide/HorizontalSlideController.swift

```diff
@@ -28,6 +28,24 @@ class HorizontalSlideController: UIPresentationController {
         return dimmingView
     }()
 
+    // MARK: - Init
+
+    /// Initializes and returns a presentation controller for transitioning between the specified view controllers with a width percentage for the container.
+    ///
+    /// - Parameters:
+    ///   - presentedViewController: The view controller being presented modally.
+    ///   - presentingViewController: The view controller whose content represents the starting point of the transition.
+    ///   - containerPercentage: The width of the container for the view controller being presented modally
+    init(presentedViewController: UIViewController, presenting presentingViewController: UIViewController?, containerPercentage: CGFloat) {
+        self.containerPercentage = containerPercentage
+        super.init(presentedViewController: presentedViewController, presenting: presentingViewController)
+    }
+
+    override init(presentedViewController: UIViewController, presenting presentingViewController: UIViewController?) {
+        self.containerPercentage = UIDevice.current.userInterfaceIdiom == .pad ? 0.60 : 0.85
```

Comment:
What if you put this in the `HorizontalSlideTransition` class instead as a default value? Then you don't really need this init and you can remove the optional below.

Reply:
That sounds like a much better idea!